core/src/main/java/org/apache/spark/memory/MemoryConsumer.java

```diff
@@ -41,7 +41,7 @@ protected MemoryConsumer(TaskMemoryManager taskMemoryManager, long pageSize, Mem
   }
 
   protected MemoryConsumer(TaskMemoryManager taskMemoryManager) {
```

Comment:
so, do we still need `MemoryConsumer(TaskMemoryManager taskMemoryManager, long pageSize, MemoryMode mode)`? I think it can just be `MemoryConsumer(TaskMemoryManager taskMemoryManager, long pageSize)`